tools/lib/provision.py

```diff
@@ -308,9 +312,9 @@ def main(options):
     else:
         # hash the content of setup-yum-repo and build-*
         sha_sum.update(open('scripts/lib/setup-yum-repo', 'rb').read())
-        build_paths = glob.glob("scripts/lib/build-")
-        for bp in build_paths:
-            sha_sum.update(open(bp, 'rb').read())
+    build_paths = glob.glob("scripts/lib/build-")
+    for bp in build_paths:
+        sha_sum.update(open(bp, 'rb').read())
```

Comment:
This seems like an unrelated logic change, perhaps fixing a bug you uncovered working on the rest?  Please separate it out as its own commit with a commit message explaining the bug (And similarly for the other changes that aren't just adding `focal` or `20.04` to a list/conditional).

Reply:
@timabbott Sorry I forgot to justify this in the commit message. Can you check that again I have separated these in different commits?